### ISSUE TITLE
Does Anakin run multi-thread in its own code? 

### ISSUE THREAD

**Issue by avinashcpandey** [PaddlePaddle/Anakin#514]
I see if i set OMP_NUM_THREADS or MKL_NUM_THREADS  Anakin runs multi thread. This multithread control threading in MKLML library not in Anakin code. Correct me if i am wrong.
Does Anakin support multi-threding in its own code?

**Comment by qq332982511** [comment]
YES, we support multi-thread in our JIT code，Convolution，LSTM, and so on, so you have to set OMP_NUM_THREADS. We also use MKL_GEMM , Tanh to build our FC and RNN op. It is also necessary to set MKL_NUM_THREADS
